WordPress/Classes/ViewRelated/Stats/Period Stats/Overview/OverviewCell.swift

```diff
@@ -168,7 +168,7 @@ private extension OverviewCell {
         ])
     }
 
-    private enum ChartBottomMargin {
+    enum ChartBottomMargin {
```

Comment:
Unless I'm blind (and my cmd-f has broken) nothing else in this PR references `ChartBottomMargin` — why the change here?

Reply:
Oh, sorry, I failed to commit this separately. I just noticed this `private` was unnecessary since this `enum` is in a private extension, so I removed it.